docker-compose.yml

```diff
@@ -10,3 +10,5 @@ services:
     container_name: redis
     image: redislabs/redistimeseries
     restart: always
```

Comment:
Can you expose port 6379 on this container so that we can connect to it using the Python code and redis-cli please?  I forgot about that!

Should be a case of adding:

```yaml
    ports:
      - 6379:6379
```